internal/databases/mongo/binary/utilities.go

```diff
@@ -66,3 +68,10 @@ func EnsureCompatibilityToRestoreMongodVersions(backupMongodVersion, restoreMong
 	}
 	return nil
 }
+
+// ComputeMongoStartTimeout compute timeout with formula max(10min, min(UncompressedSizeTb, 1) * 40min)
+func ComputeMongoStartTimeout(backupUncompressedSize int64) time.Duration {
+	var tb float64 = 1 << (10 * 4)
+	coef := math.Min(float64(backupUncompressedSize)/tb, 1)
+	return time.Duration(math.Max(10, coef*40) * float64(time.Minute))
```

Comment:
Why coef = min(..., 1)? what if 1.5 Tb? we need more time for 1.5 Tb
And time.Duration is int64. what do you think about:
```
dbSizeInTb := float64(backupUncompressedSize) / math.Pow(2, 40)
estimatedMinutes := dbSizeInTb * 40  // it is heuristics - mongo need about 40 minutes for 1 Tb db size
return time.Duration(math.Max(10, estimatedMinutes)) * time.Minute
```

Reply:
I guess that 40 minutes is enough to start mongo , so I set the limit on top to 40 minutes .

I know that the time.Duration is int64 . The time.Minute is nanos , so after multiplying and cast to the Duration we can discard couple of nanos, but for this timeout it is a minor problem